pkg/apic/client.go

```diff
@@ -683,27 +686,29 @@ func (c *ServiceClient) CreateSubResourceScoped(
 	wg := &sync.WaitGroup{}
 
 	for subName, sub := range subs {
-		wg.Add(1)
+		if strings.HasPrefix(subName, "x-") {
+			wg.Add(1)
 
-		base := c.cfg.GetURL()
-		url := fmt.Sprintf("%s/apis/%s/%s/%s/%s/%s/%s/%s", base, group, version, scopeKindPlural, scopeName, resKindPlural, name, subName)
+			base := c.cfg.GetURL()
+			url := fmt.Sprintf("%s/apis/%s/%s/%s/%s/%s/%s/%s", base, group, version, scopeKindPlural, scopeName, resKindPlural, name, subName)
```

Comment:
Not that it is needed for this story, but wonder if we can utilize the registry in the apiserver/v1 package to assist in creating this URL

Reply:
That would be interesting. That was made for the resource specific clients. Not sure what it would take to start using that.